sklearn/datasets/kddcup99.py

```diff
@@ -347,6 +345,21 @@ def _fetch_brute_kddcup99(subset=None, data_home=None,
 
         joblib.dump(X, samples_path, compress=0)
         joblib.dump(y, targets_path, compress=0)
+
+        # check md5 of dumped samples and targets
```

Comment:
Hmm... I'm not sure if there's any guarantee about the consistency of a joblib dumped object's binary representation, from run to run or from version to version.